Wegli/App/Redux/AppState.swift

```diff
@@ -15,13 +15,36 @@ import UIKit
 struct Report {
     var images: [UIImage] = []
     var address: CNPostalAddress?
+    
+    // MARK: Description
+    struct Car {
+        var color: String?
+        var type: String?
+        var licensePlateNumber: String?
+    }
+    var car = Car()
+    
+    struct Crime {
+        static let crimes = ["Stand auf der Radspur", "Core Data", "Core Data", "Core Data"]
```

Comment:
@phoet I could use some input on the input values. Is there something I could use already in the old app?

Reply:
https://github.com/weg-li/weg-li/blob/master/app/lib/vehicle.rb#L165-L212

Reply:
It's not a crime btw

Reply:
haha. I am open for better naming. But thanks for the hint

Reply:
I use `charges` 👍

Reply:
@phoet is there also a reference on how the email looks like?

Reply:
Of course, but the easiest way is to use weg-li and look at the preview.